package.json

```
@@ -1686,6 +1697,11 @@
                 "key": "ctrl+alt+c p",
                 "when": "editorLangId == clojure && calva:keybindingsEnabled"
             },
+            {
+                "command": "calva.toggleEvaluationSendCodeToOutputWindow",
+                "key": "ctrl+alt+o ctrl+s",
```

I think it is easier on the fingers to not have ctrl for the second stroke. (And also your comment suggested this might be a mistake. 😄 

```suggestion
                "key": "ctrl+alt+o s",
```